ui/page/settings/app_settings_page.go

```diff
@@ -343,6 +343,10 @@ func (pg *AppSettingsPage) networkSettings() layout.Widget {
 
 func (pg *AppSettingsPage) dexSettings() layout.Widget {
 	return func(gtx C) D {
+		if pg.AssetsManager.NetType() == libutils.Mainnet {
+			return D{}
+		}
```

Comment:
Won't this show a "DEX" section with nothing in it?

I think we'd get a similar display for testnet if dex isn't yet initialized with a password.

Reply:
> Won't this show a "DEX" section with nothing in it? 

No, but you are right  about this "I think we'd get a similar display for testnet if dex isn't yet initialized with a password.". The checks should come early.

Reply:
Updated